lib/bitreserve/entities/base_entity.rb

```diff
@@ -3,15 +3,25 @@ module Entities
     class BaseEntity
       include Virtus.model
 
-      def self.from_collection(entities)
-        entities.map do |entity|
-          new(entity)
-        end
+      def self.from_collection(entities, content_range)
```

Comment:
Cool. Thanks. :)

@naps62 I think it's best to leave it optional, since not all collections will be paginated. What do you think? Should I go on with this approach?